sv2/parsers-sv2/src/lib.rs

```diff
@@ -23,6 +23,13 @@
 //!   submission).
 
 pub mod error;
+pub mod tlv;
+pub mod tlv_extensions;
```

Comment:
We are already exposing public facing API from these modules.
```suggestion
mod tlv;
mod tlv_extensions;
```